src/loaders/LoaderUtils.js

```diff
@@ -1,36 +1,10 @@
 class LoaderUtils {
 
-	static decodeText( array ) {
+	static decodeText( array ) { // @deprecated, r165
 
-		if ( typeof TextDecoder !== 'undefined' ) {
+		console.warn( 'THREE.LoaderUtils: decodeText() has been deprecated with r165 and will be removed with r175. Use TextDecoder instead.' );
 
-			return new TextDecoder().decode( array );
```

Comment:
The code is still in place with deprecated methods.

Reply:
good